components/01-secrets/README.md

```diff
@@ -121,6 +121,44 @@ for skrt in $(find . -name "secret-keystone*.yaml" -depth 1); do
 done
 ```
 
+## Ironic
+
+Generate the necessary secrets for OpenStack Ironic.
+
+```bash
+kubectl --namespace openstack \
+        create secret generic ironic-rabbitmq-password \
+        --type Opaque \
+        --from-literal=username="ironic" \
+        --from-literal=password="$($(git rev-parse --show-toplevel)/scripts/pwgen.sh)" \
+        --dry-run -o yaml > secret-ironic-rabbitmq-password.yaml
+kubectl --namespace openstack \
+        create secret generic ironic-db-password \
+        --type Opaque \
+        --from-literal=password="$($(git rev-parse --show-toplevel)/scripts/pwgen.sh)" \
+        --dry-run -o yaml > secret-ironic-db-password.yaml
+kubectl --namespace openstack \
+        create secret generic ironic-keystone-password \
+        --type Opaque \
+        --from-literal=username="ironic" \
+        --from-literal=password="$($(git rev-parse --show-toplevel)/scripts/pwgen.sh)" \
+        --dry-run -o yaml > secret-ironic-keystone-password.yaml
+```
+
+Now let's seal them.
+
+```bash
+for skrt in $(find . -name "secret-ironic*.yaml" -depth 1); do
```

Comment:
as per the other PR - this probably should  be `-maxdepth 1` ? 
```
❯ find . -name "secret-ironic*.yaml" -depth 1
find: warning: you have specified the global option -depth after the argument -name, but global options are not positional, i.e., -depth affects tests specified before it as well as those specified after it.  Please specify global options before other arguments.
find: paths must precede expression: `1'
```

the `-depth` is just [a boolean flag](https://man.cx/find#heading5)

Reply:
You're right. I copied this before you fixed it in the file I copied it from. Good catch.

Reply:
Found where you fixed this before... 57ad0553d48